python/pyproject.toml

```diff
@@ -26,7 +26,8 @@ runtime_common = [
 srt = [
     "sglang[runtime_common]", "cuda-python",
     "sgl-kernel>=0.0.3.post5", "torch", "vllm>=0.6.4.post1,<=0.7.2",
-    "flashinfer_python>=0.2.0.post2", "outlines>=0.0.44,<=0.1.11"
+    "flashinfer_python @ https://github.com/flashinfer-ai/flashinfer-nightly/releases/download/0.2.1%2Bdbb1e4e/flashinfer_python-0.2.1+dbb1e4e.cu124torch2.5-cp310-cp310-linux_x86_64.whl",
```

Comment:
This is only for testing purposes as the new release is still in progress. https://github.com/flashinfer-ai/flashinfer/actions/runs/13307454841